docs/samples/Microsoft.ML.Samples.FSharp/Dynamic/Transforms/Text/ApplyCustomWordEmbedding.fs

```diff
@@ -0,0 +1,79 @@
+﻿module Microsoft.ML.Samples.Dynamic.ApplyCustomWordEmbeddings
+open Microsoft.ML
+open Microsoft.ML.Data
+open System
+open System.IO
+
+type TextData(value:string) = 
+    member val Text = value with get,set
+    new() = TextData("")
+
+type TransformedTextData(value: string) = 
+    inherit TextData(value)
+    member val Features:float32 array = Array.empty with get, set
```

Comment:
`member val Features : float32 [] = Array.empty with get, set` is the preferred type annotation for arrays